api/test/integration/test_manager_endpoints.tavern.yaml

```diff
@@ -677,11 +675,10 @@ stages:
             - wazuh-modulesd:control: !anything
             - wazuh-modulesd:database: !anything
             - wazuh-modulesd:download: !anything
-            - wazuh-modulesd:oscap: !anything
             - wazuh-modulesd:osquery: !anything
             - wazuh-modulesd:syscollector: !anything
           failed_items: []
-          total_affected_items: 23
+          total_affected_items: 21
```

Comment:
Change this test to check for anything in the affected_items and any int in the total_affected_items